WcaOnRails/.rubocop.yml

```diff
@@ -178,3 +176,10 @@ Naming/PredicateName:
 
 Style/YodaCondition:
   Enabled: false
+
+Style/FormatStringToken:
+  Enabled: false
```

Comment:
Rubocop doesn't like our format strings https://github.com/thewca/worldcubeassociation.org/blob/37dacc53336c89949728d628725135c2b0ae69d7/WcaOnRails/lib/solve_time.rb#L8
It prefers something like this
```ruby
CLOCK_FORMAT = "%<hours>d:%<minutes>02d:%<seconds>02d.%<centis>02d"
format(CLOCK_FORMAT, hours: hours, minues: minutes, seconds: seconds, centis: centis)
```
Looks quite redundant to me (at least in this particular case), but I don't have strong opinion. So if you like the rule I can just modify the code to satisfy it =)